eosetl/mappers/transaction_input_mapper.py

```diff
@@ -0,0 +1,86 @@
+# MIT License
+#
+# Copyright (c) 2018 Omidiora Samuel, samparsky@gmail.com
```

Comment:
Update licence in all files @Vasiliy-Bondarenko

Reply:
done

Reply:
don't know what to do about /setup.py